packages/react-canvas-core/src/states/DragCanvasState.ts

```diff
@@ -46,6 +46,13 @@ export class DragCanvasState extends AbstractDisplacementState {
 	}
 
 	fireMouseMoved(event: AbstractDisplacementStateEvent) {
+		if (!event.isMouseDown) {
```

Comment:
so what I actually meant, was that we might want to check for this before we call `fireMouseMoved` since all states which extend this AbstractDisplacementState will suffer from the same problem. As in make AbstractDisplacementState do the check

Reply:
Done